particles/scenario.cpp

```diff
@@ -22,6 +22,12 @@
 #include <random>
 #include <iostream>
 #include "scenario.h"
+#include "field.h"
+
+#define cross_product(av,bv) (av).cross(bv)
+#define dot_product(av,bv) (av).dot(bv)
+#define vector_length(v) (v).norm()
+#define normalize_vector(v) (v).normalized()
```

Comment:
These are already defined in relativistic_math.h

Reply:
I simply used these defines as a drop-in replacement for vectorclass. I originally took them from sysboundaries/ionosphere.cpp

So... maybe a global Eigen-as-vectorclass-header would make sense?